es-ds-docs/components/ds-molecules-list.vue

```diff
@@ -33,6 +33,9 @@
         <li>
             <ds-link to="/molecules/form-message"> Form message </ds-link>
         </li>
+        <li>
+            <ds-link to="/molecules/form-textarea"> Form textarea </ds-link>
```

Comment:
Wondering why this is "Form textarea" instead of "Textarea" as the old one was?

Reply:
Good question - I was matching the component name but could get behind just putting Textarea